package-scripts/modules/ajexport.js

```diff
@@ -0,0 +1,64 @@
+// With thanks to elenterius on discord for troubleshooting
+// https://discord.com/channels/314078526104141834/1189404550986211329/1189517519262855229
+
+/* global Project, loadModelFile, AnimatedJava */
+
+const { readdirSync, readFileSync } = require('fs');
+
+export async function script() {
+  if (typeof AnimatedJava === 'undefined') {
+    throw new Error('Failed to load Animated Java plugin before CLI plugin');
+  }
+  const paths = getConfigPaths('./package-scripts/modules/config.json');
+  const modelDir = paths.ajmodelPath.concat('/');
+  console.log('Target paths: ', paths);
+  const files = readdirSync(modelDir).filter((file) =>
+    file.includes('ajmodel'),
```

Comment:
```suggestion
    file.endsWith('.ajmodel'),
```

effectively same behavior, slightly more correct + readable

technically fixes a bug if we happened to have `ajmodel.txt` as a file which would pass and break the script (assuming it wasn't a real model file)